src/main/java/io/leonis/zosma/exception/RethrowExceptionBiConsumer.java

```diff
@@ -0,0 +1,35 @@
+package io.leonis.zosma.exception;
+
+import lombok.Value;
+
+/**
+ * The Class RethrowExceptionBiConsumer.
+ *
+ * This class represents a {@link BiConsumerWithExceptions} which rethrows
+ * {@link Exception Exceptions} as {@link RuntimeException RuntimeExceptions}.
+ *
+ * @param <T> The type of the first argument passed to the {@link BiConsumerWithExceptions}.
+ * @param <U> The type of the second argument passed to the {@link BiConsumerWithExceptions}.
+ * @param <E> The type of exception which may be thrown by the {@link BiConsumerWithExceptions}.
+ * @author Rimon Oz
+ * @author Jeroen de Jong
+ */
+@Value
+public class RethrowExceptionBiConsumer<T, U, E extends Exception> {
```

Comment:
shouldn't this `extend BiConsumer<T, U>`?

Reply:
That makes total sense. I'll add the extensions of the native types in a minute.